inlang/source-code/versioned-interfaces/message-bundle/src/interface.ts

```diff
@@ -0,0 +1,115 @@
+import { LanguageTag as Locale } from "@inlang/language-tag"
+import { Type, type Static } from "@sinclair/typebox"
+
+export type Literal = Static<typeof Literal>
+export const Literal = Type.Object({
+	type: Type.Literal("literal"),
+	value: Type.String(),
+})
+
+/**
+ * A (text) element that is translatable and rendered to the UI.
+ */
+export type Text = Static<typeof Text>
+export const Text = Type.Object({
+	type: Type.Literal("text"),
+	value: Type.String(),
+})
+
+export type VariableReference = Static<typeof VariableReference>
+export const VariableReference = Type.Object({
+	type: Type.Literal("variable"),
```

Comment:
Mismatch between type name and type string `VariableReference` vs `variable`. Recommendation, follow MessageFormat PascalCase for all types: 

```diff
-variable
+VariableRefernece
```

Reply:
The string `"variable"` is from the MessageFormat 2.0 Data-model recommendation
https://github.com/unicode-org/message-format-wg/blob/6d7b4ba213e686ff2d403d3025d38d76b42b75f7/spec/data-model/README.md#expressions

Reply:
Oh wow, they deviated from the previous pattern then. I let @jldec decide. I prefer written out types it's a naming thing though which no real impact